config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-4-upgrades.yaml

```diff
@@ -46,17 +45,20 @@ periodics:
         resources:
           requests:
```

Comment:
Looks like we also have to drop the serviceAccountName: xref: https://github.com/kubernetes/test-infra/pull/30344/files